src/opesciIO.cpp

```diff
@@ -57,453 +57,646 @@
+        //                                   3 = exponential
+        //                                   4 = other
+        // 3241 - 3242 	Trace number of sweep channel.
+        // std::cout<<"3241 - 3242 	Trace number of sweep channel: "<<unpack_int16(header_buffer+2340, swap_endian)<<std::endl;
+
+        // 3243 - 3244 	Sweep trace taper length, ms, at start if tapered.
+        // std::cout<<"3243 - 3244 	Sweep trace taper length, ms, at start if tapered: "<<unpack_int16(header_buffer+2342, swap_endian)<<std::endl;
+
+        // 3245 - 3246 	Sweep trace taper length, ms, at end.
+        // std::cout<<"3245 - 3246 	Sweep trace taper length, ms, at end: "<<unpack_int16(header_buffer+2344, swap_endian)<<std::endl;
+
+        // 3247 - 3248 	Taper type: 1 = linear 2 = cos 3 = other
+        // std::cout<<"3247 - 3248 	Taper type: 1 = linear 2 = cos 3 = other: "<<unpack_int16(header_buffer+2346, swap_endian)<<std::endl;
+
+        // 3249 - 3250 	Correlated data traces: 1 = no 2 = yes
+        // std::cout<<"3249 - 3250 	Correlated data traces: 1 = no 2 = yes"<<unpack_int16(header_buffer+2348, swap_endian)<<std::endl;
+
+        // 3251 - 3252 	Binary gain recovered: 1 = yes 2 = no
+        // std::cout<<"3251 - 3252 	Binary gain recovered: 1 = yes 2 = no: "<<unpack_int16(header_buffer+2350, swap_endian)<<std::endl;
+
+        // 3253 - 3254 	Amplitude recovery method: 1 = none 2 = spherical divergence 3 = AGC 4 = other
+        // std::cout<<"3253 - 3254 	Amplitude recovery method: 1 = none 2 = spherical divergence 3 = AGC 4 = other: "<<unpack_int16(header_buffer+2352, swap_endian)<<std::endl;
+
+        // 3255 - 3256 	Measurement system: 1 = meters 2 = feet
+        // std::cout<<"3255 - 3256 	Measurement system: 1 = meters 2 = feet: "<<unpack_int16(header_buffer+2354, swap_endian)<<std::endl;
+
+        // 3257 - 3258 	Impulse signal: 1 = Upward = negative number.
+        //                                  2 = Upward = positive number.
+        // std::cout<<"3257 - 3258 	Impulse signal: "<<unpack_int16(header_buffer+2356, swap_endian)<<std::endl;
+
+        // 3259 - 3260 	Vibratory polarity code - seismic signal lags pilot signal by: 1 = 337.5 - 22.5 degrees
+        //                                                                                 2 = 22.5 - 67.5 degrees
+        //                                                                                 3 = 67.5 - 112.5 degrees
+        //                                                                                 4 = 112.5 - 157.5 degrees
+        //                                                                                 5 = 157.5 - 202.5 degrees
+        //                                                                                 6 = 202.5 - 247.5 degrees
+        // std::cout<<"3259 - 3260 	Vibratory polarity code - seismic signal lags pilot signal by: "<<unpack_int16(header_buffer+2358, swap_endian)<<std::endl;
```

Comment:
I'm not sure if all the comments in this function are documentation or debug leftovers. If it's the latter please remove before merging.